torch/_inductor/select_algorithm.py

```diff
@@ -1176,8 +1176,11 @@ def get_inputs():
             )
             expected = None
             if VERIFY:
-                choices[0].benchmark(*example_inputs_extern, out=out_extern)
-                expected = out_extern.clone()
+                for i in range(len(choices)):
```

Comment:
The way I understand it, in that case no "expected" value is returned. Since this is just code that supports unit test ( e.g. produce a reference result to compare against), the unit test that runs this should be responsible for producing a meaningful assertion failure.

See test_select_algorithm.py - this code only gets triggered by the tests in that file, which patch the VERIFY object here: https://github.com/pytorch/pytorch/blob/4b586a434f136f987513fa64599f281b679607a4/test/inductor/test_select_algorithm.py#L30